client/hosting-overview/components/active-domains-card.tsx

```diff
@@ -0,0 +1,44 @@
+import { Button, Card } from '@automattic/components';
+import { useSiteDomainsQuery } from '@automattic/data-stores';
+import { DomainsTable } from '@automattic/domains-table';
+import { useTranslate } from 'i18n-calypso';
+import { FC } from 'react';
+import { fetchSiteDomains } from 'calypso/my-sites/domains/domain-management/domains-table-fetch-functions';
+import { useSelector } from 'calypso/state';
+import { getSelectedSite } from 'calypso/state/ui/selectors';
+
+const ActiveDomainsCard: FC = () => {
+	const site = useSelector( getSelectedSite );
+	const { data, isLoading } = useSiteDomainsQuery( site?.ID, {
+		queryFn: () => fetchSiteDomains( site?.ID ),
+	} );
+	const translate = useTranslate();
+
+	return (
+		<Card className="hosting-overview__card">
+			<div className="hosting-overview__card-header">
+				<h3 className="hosting-overview__card-title">{ translate( 'Active domains' ) }</h3>
+
+				<Button className="hosting-overview__link-button" plain href="/start/domain">
+					{ translate( 'Add new domain' ) }
+				</Button>
+				<Button
+					className="hosting-overview__link-button"
+					plain
+					href={ `/domains/manage/${ site?.slug }` }
+				>
+					{ translate( 'Manage domains' ) }
+				</Button>
+			</div>
+
+			<DomainsTable
+				isLoadingDomains={ isLoading }
+				domains={ data?.domains }
+				isAllSitesView={ false }
+				siteSlug={ site?.slug ?? null }
+			/>
```

Comment:
Can we remove the "Add site" link rendered by this component?

<img width="269" alt="Screenshot 2024-04-12 at 14 21 41" src="https://github.com/Automattic/wp-calypso/assets/1233880/7de352df-21aa-47fb-b22d-c7a498431a78">

It feels an off place for that link, and it does not appear on the Figma designs

Reply:
I don't think I've seen it while testing 🤔 Which kind of site/plan are you using in order to see it?

Reply:
It's a simple site with a Free plan

Reply:
That list surely has some wonkiness going on.

On my main account, which has a Simple site on Creator without custom domains, I get the ***.wordpress.com primary domain without "Add site" link.

Using a few test accounts, with primary Simple/Free sites, I get an empty list:

<img width="1141" alt="Screenshot 2024-04-15 at 09 44 16" src="https://github.com/Automattic/wp-calypso/assets/2070010/0fff154e-29bd-4665-bda6-9f74e9a21752">

I presume my main account has cached stuff, and this needs some query component to fetch the data? 🤔 

---

Also, interesting how I'm getting the [exact same content](https://github.com/Automattic/wp-calypso/pull/89476#issuecomment-2051646921) as Miguel.

Reply:
> Using a few test accounts, with primary Simple/Free sites, I get an empty list:

You're probably experiencing the issue of a site not being selected yet in the Redux store